z3c/dependencychecker/package.py

```diff
@@ -47,10 +47,14 @@ def setup_py_path(self):
     @cached_property
     def package_dir(self):
         """Check where the .egg-info is located"""
-        try_paths = (
-            self.distribution_root,
-            os.path.join(self.distribution_root, 'src', )
-        )
+        try_paths = [
+            self.distribution_root
+        ]
+        try_paths += [
+            '{0}/{1}'.format(self.distribution_root, folder)
+            for folder in os.listdir(self.distribution_root)
+            if os.path.isdir('{0}/{1}'.format(self.distribution_root, folder))
```

Comment:
I generally want to do the os.path.join just once:

    toplevel_items = [...join... for item in os.listdir(...)]
    try_paths += [folder for folder in toplevel_items if os.path.isdir(folder)]

That way you don't have a duplicate in the "if". But that's just a style thingy. Having just one list comprehension also is ok.